pom.xml

```diff
@@ -79,21 +81,39 @@
   </distributionManagement>
 
   <properties>
-    <checkstyle.version>6.1.1</checkstyle.version>
+    <maven.compiler.source>1.6</maven.compiler.source>
```

Comment:
What for ist that? As we allow Java 8 for  Hawkular in general, do we need to restrict to 6 here?